### ISSUE TITLE
VREF  does not work when INPUT_PULLUP is used?

### ISSUE THREAD

**Issue by ole00** [mrjimenez/JTAG#13]
I think the vref detection can't work properly on its own if the INPUT_PULLUP is used on  the vref pin. When the JTAG cable is disconnected then the internal pull-up on the VREF pin will cause the vref detection to always read logical one - JTAG cable connected.  I don't think there is a good solution for this, unless an external pull down resistor (about 2k) is used as well. Changing the INPUT_PULLUP to INPUT (without pull-up) helps a bit, but because the VREF pin is then left floating it might also trigger wrong vref detection depending on the electrical noise on the vref wire. Perhaps the best would be add an option to ignore vref completely if a value of 0 (or -1) is specified as VREF pin when the player is constructed.

**Comment by mrjimenez** [comment]
Indeed, I don't think there is a good solution without extra hardware.
